subprojects/launcher/src/main/java/org/gradle/launcher/daemon/protocol/InvalidateFileSystemLocations.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.launcher.daemon.protocol;
+
+import java.util.List;
+import java.util.UUID;
+
+public class InvalidateFileSystemLocations extends Command {
```

Comment:
We could also call this `InvalidateVirtualFileSystem` like we already have a cross-version test by this name.

Reply:
Done.